contracts/src/v0.8/shared/token/ERC677/BurnMintERC677.sol

```diff
@@ -1,217 +1,33 @@
-// SPDX-License-Identifier: MIT
```

Comment:
Why are we changing this?

Reply:
I thought you said you wanted it at BUSL license?